src/_common/__tests__/Tab-test.js

```diff
@@ -19,6 +19,16 @@ describe('<Tab />', () => {
         expect(wrapper.text()).to.equal('');
     });
 
+    it('should render tab with close button', () => {
```

Comment:
Tests. Good.
Be more descriptive. This is almost like documentation.
Good text here allows us to not write comments in the actual source code.
In that case I might call it 'renders a close button when showCloseIcon is true'.